views/survey.pug

```diff
@@ -21,7 +21,7 @@ html
               else if survey[index-1].group_prompt != item.group_prompt
                 h2= item.group_prompt ? item.group_prompt : generic_group_prompt
               if item.prompt
-                li(class='inputBlock')
+                li(class='inputBlock')= start++

```

Comment:
might be better to put this number as part of the h4 so it doesn't wrap lines. Also trail with a `.`, so it looks like a number. 

e.g., 
> 3. Select the word that best describes what the person in the picture is thinking or feeling